experimental/traffic-portal/src/app/core/certs/cert-viewer/cert-viewer.component.scss

```diff
@@ -0,0 +1,33 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+.tab-content {
+	margin-top: 5px;
+	display: grid;
+
+	&:is(form), .mat-accordion {
+		width: 98%;
+		margin: auto auto 15px;
+	}
+}
+
+.private-text {
```

Comment:
I looked into the styling on this, and the issue is that the Angular Material styling is more specific because it excludes `disabled` controls. So all you have to do is change this selector to match that specificity - however, we don't want it to not handle disabled textareas, since those can still be private. So just split the styling up to be more explicit:
```scss
.private-text {
	&:not(:focus) {
		color: transparent;
		text-shadow: 0 0 8px rgba(0,0,0,0.5);
	}

	&:focus {
		color: inherit;
		text-shadow: none;
	}
}
```
that's like one extra line, but doesn't require `!important`. If you wanna argue that that's somehow worse than just using `!important`, I'll hear you out, but just note that it can be done.

Reply:
I stand corrected